sbt-plugin-test/build.sbt

```diff
@@ -182,10 +182,30 @@ lazy val jsDependenciesTest = project.settings(versionSettings: _*).
           "compressed/history.js appears before foo.js")
 
       streams.value.log.info("jsDependencies resolution test passed")
+
+      // Regression test for issue #2243
+      val pack = (packageJSDependencies in Compile).value
+      val minPack = (packageMinifiedJSDependencies in Compile).value
```

Comment:
Also I would encourage using `Def.sequential` to encourage sbt not to run both this things in parallel, since otherwise the test could accidentally work due to a race.